2023-09-01 16:14:05,275 ERROR 71297 [-/127.0.0.1/-/95ms GET /api/common/index/console] ConfigUtilService.get error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:14:05,280 ERROR 71297 [-/127.0.0.1/-/100ms GET /api/common/index/console] ConfigUtilService.getVal error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:14:05,281 ERROR 71297 [-/127.0.0.1/-/101ms GET /api/common/index/console] IndexService.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:14:05,281 ERROR 71297 [-/127.0.0.1/-/101ms GET /api/common/index/console] IndexController.index error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:14:48,365 ERROR 71297 [-/127.0.0.1/-/35ms GET /api/common/index/console] ConfigUtilService.get error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:14:48,365 ERROR 71297 [-/127.0.0.1/-/35ms GET /api/common/index/console] ConfigUtilService.getVal error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:14:48,365 ERROR 71297 [-/127.0.0.1/-/35ms GET /api/common/index/console] IndexService.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:14:48,365 ERROR 71297 [-/127.0.0.1/-/35ms GET /api/common/index/console] IndexController.index error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:15:00,217 ERROR 71297 [-/127.0.0.1/-/34ms GET /api/common/index/console] ConfigUtilService.get error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:15:00,217 ERROR 71297 [-/127.0.0.1/-/34ms GET /api/common/index/console] ConfigUtilService.getVal error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:15:00,217 ERROR 71297 [-/127.0.0.1/-/34ms GET /api/common/index/console] IndexService.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:15:00,217 ERROR 71297 [-/127.0.0.1/-/34ms GET /api/common/index/console] IndexController.index error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:15:38,977 ERROR 71439 [-/127.0.0.1/-/38ms GET /api/common/index/console] IndexController.index error: TypeError: this.console.log is not a function
2023-09-01 16:18:07,326 ERROR 71707 [-/127.0.0.1/-/35ms GET /api/common/index/console] IndexController.index error: TypeError: this.console.log is not a function
2023-09-01 16:20:21,278 ERROR 71951 [-/127.0.0.1/-/47ms GET /api/common/index/console] systemController.console error: TypeError: this.console.log is not a function
2023-09-01 16:22:24,253 ERROR 72115 [-/127.0.0.1/-/38ms GET /api/common/index/console] systemController.console error: TypeError: this.console.log is not a function
2023-09-01 16:22:48,597 ERROR 72164 [-/127.0.0.1/-/37ms GET /api/common/index/console] systemController.console error: TypeError: this.console.log is not a function
2023-09-01 16:23:14,237 ERROR 72213 [-/127.0.0.1/-/33ms GET /api/common/index/console] systemController.console error: TypeError: this.console.log is not a function
2023-09-01 16:25:42,560 ERROR 72463 [-/127.0.0.1/-/38ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:26:46,010 ERROR 72562 [-/127.0.0.1/-/41ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:27:28,687 ERROR 72624 [-/127.0.0.1/-/34ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:28:12,521 ERROR 72699 [-/127.0.0.1/-/31ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:28:40,589 ERROR 72744 [-/127.0.0.1/-/12ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:31:48,567 ERROR 72991 [-/127.0.0.1/-/75ms GET /api/common/index/console] ConfigUtilService.get error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:31:48,571 ERROR 72991 [-/127.0.0.1/-/79ms GET /api/common/index/console] ConfigUtilService.getVal error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:31:48,572 ERROR 72991 [-/127.0.0.1/-/80ms GET /api/common/index/console] IndexService.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:31:48,572 ERROR 72991 [-/127.0.0.1/-/80ms GET /api/common/index/console] systemController.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:32:23,032 ERROR 73061 [-/127.0.0.1/-/35ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:32:40,498 ERROR 73103 [-/127.0.0.1/-/19ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:32:58,661 ERROR 73137 [-/127.0.0.1/-/48ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:33:18,576 ERROR 73171 [-/127.0.0.1/-/29ms GET /api/common/index/console] systemController.console error: TypeError: Cannot read properties of undefined (reading 'log')
2023-09-01 16:33:44,829 ERROR 73218 [-/127.0.0.1/-/42ms GET /api/common/index/console] ConfigUtilService.get error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:33:44,829 ERROR 73218 [-/127.0.0.1/-/42ms GET /api/common/index/console] ConfigUtilService.getVal error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:33:44,829 ERROR 73218 [-/127.0.0.1/-/42ms GET /api/common/index/console] IndexService.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:33:44,829 ERROR 73218 [-/127.0.0.1/-/42ms GET /api/common/index/console] systemController.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:34:07,421 ERROR 73254 [-/127.0.0.1/-/51ms GET /api/common/index/console] ConfigUtilService.get error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:34:07,421 ERROR 73254 [-/127.0.0.1/-/52ms GET /api/common/index/console] ConfigUtilService.getVal error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:34:07,421 ERROR 73254 [-/127.0.0.1/-/52ms GET /api/common/index/console] IndexService.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:34:07,421 ERROR 73254 [-/127.0.0.1/-/52ms GET /api/common/index/console] systemController.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:35:27,374 ERROR 73355 [-/127.0.0.1/-/52ms GET /api/common/index/console] ConfigUtilService.get error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:35:27,374 ERROR 73355 [-/127.0.0.1/-/52ms GET /api/common/index/console] ConfigUtilService.getVal error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:35:27,374 ERROR 73355 [-/127.0.0.1/-/52ms GET /api/common/index/console] IndexService.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:35:27,374 ERROR 73355 [-/127.0.0.1/-/52ms GET /api/common/index/console] systemController.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:36:16,390 ERROR 73450 [-/127.0.0.1/-/35ms GET /api/common/index/console] ConfigUtilService.get error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:36:16,390 ERROR 73450 [-/127.0.0.1/-/36ms GET /api/common/index/console] ConfigUtilService.getVal error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:36:16,390 ERROR 73450 [-/127.0.0.1/-/36ms GET /api/common/index/console] IndexService.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:36:16,390 ERROR 73450 [-/127.0.0.1/-/36ms GET /api/common/index/console] systemController.console error: SequelizeDatabaseError: Unknown column 'createTime' in 'field list'
2023-09-01 16:37:32,690 ERROR 73548 [-/127.0.0.1/-/51ms GET /api/common/index/console] IndexService.console error: ReferenceError: Cannot access 'version' before initialization
2023-09-01 16:37:32,690 ERROR 73548 [-/127.0.0.1/-/51ms GET /api/common/index/console] systemController.console error: ReferenceError: Cannot access 'version' before initialization
2023-09-01 16:38:26,702 ERROR 73619 [-/127.0.0.1/-/77ms GET /api/common/index/console] IndexService.console error: TypeError: now.setDate(...).toISOString is not a function
2023-09-01 16:38:26,702 ERROR 73619 [-/127.0.0.1/-/77ms GET /api/common/index/console] systemController.console error: TypeError: now.setDate(...).toISOString is not a function
2023-09-01 17:12:14,024 ERROR 76020 [-/127.0.0.1/-/15ms GET /api/common/index/config] ConfigUtilService.get error: Error: WHERE parameter "name" has invalid "undefined" value
2023-09-01 17:12:14,024 ERROR 76020 [-/127.0.0.1/-/15ms GET /api/common/index/config] IndexService.config error: Error: WHERE parameter "name" has invalid "undefined" value
2023-09-01 17:12:14,024 ERROR 76020 [-/127.0.0.1/-/15ms GET /api/common/index/config] systemController.config error: Error: WHERE parameter "name" has invalid "undefined" value
2023-09-01 17:12:53,844 ERROR 76135 [-/127.0.0.1/-/16ms GET /api/common/index/config] ConfigUtilService.get error: Error: WHERE parameter "name" has invalid "undefined" value
2023-09-01 17:12:53,844 ERROR 76135 [-/127.0.0.1/-/17ms GET /api/common/index/config] IndexService.config error: Error: WHERE parameter "name" has invalid "undefined" value
2023-09-01 17:12:53,844 ERROR 76135 [-/127.0.0.1/-/17ms GET /api/common/index/config] systemController.config error: Error: WHERE parameter "name" has invalid "undefined" value
2023-09-01 18:57:56,191 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 601173ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 18:59:33,033 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 76404ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 19:03:29,032 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 142737ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 19:06:08,068 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 138670ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 19:08:43,087 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 133922ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 19:19:18,201 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 576530ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 19:24:36,336 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 236174ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 19:26:50,072 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 93144ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 21:08:45,071 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 6094623ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-01 22:19:10,044 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 4224954ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

2023-09-02 00:07:12,245 ERROR 71679 nodejs.ClusterClientNoResponseError: client no response in 6481926ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/laijiahua 1/likeadmin/egg/node_modules/cluster-client/lib/leader.js:79:25)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
name: "ClusterClientNoResponseError"
pid: 71679
hostname: MacBook-Pro-6.local

